### ISSUE TITLE
Moderately prioritize results from Planet Drupal blogs?

### ISSUE THREAD

**Issue by rszrama** [kevinquillen/drupal-goggles#1]
The challenge here is a lot the blogs are indeed old, not updated, and oriented around Drupal 5, 6, and 7 content. Perhaps a moderate bump, 1.5x, but manually review the list of blogs to exclude those that haven't posted relevant content in the past 3 or 4 years?

**Comment by kevinquillen** [comment]
Good idea. Is there a quick way to get a list of them to add to the file and score them?
